pkg/feeds/sequence/sequence.go

```diff
@@ -158,7 +158,7 @@ func (f *asyncFinder) At(ctx context.Context, at, after int64) (ch swarm.Chunk,
 		return nil, nil, &index{uint64(after)}, nil
 	}
 	// if chunk exists construct an initial interval with base=0
-	c := make(chan *result)
+	c := make(chan *result, 32)
```

Comment:
I'm not sure about this change, could you perhaps provide some reasoning as to how this buffered channel solves the goroutine leak and how the number 32 was derived?

Reply:
The reason to use buffered channel here is similar as described below. 

In this case there many producer gorutines `f.at` which will wait to write to channel `c`.  If we wanted to solve this without buffered channel,  lines 229 and 231 would require to have select statements with context signal. 

Channel capacity is irrelevant, but should be bit larger because there could be multiple results writing simultaneously.